### PR TITLE
Update irasa.md

### PR DESCRIPTION
**Pull request by ruiliu-psy**
1) adding subtitles to the ECoG section.
2) adding citation of the Wen&Liu 2016 and Stolk et al. 2019.
3) removing the statement 'consistent with the original study' and keeping the interpretation of the results. I understand you may want to claim that the consistency of the results getting from the previous and current implementation, but the word 'original' sounds like referring to Wen&Liu 2016 and if people had read Stolk et al. 2019 they should already now what the 'original' results was; if not the statement would be out of blue to them.
4) changing Arjen's summary about the new implementation, as it was not the purpose of the update.
5) removing plots of the new vs. old implementation.

### CODE REVIEW

**Comment by ruiliu-psy** [comment]
@StolkArjen ?

**Comment by StolkArjen** [comment]
Thanks, Rui;

#IRASA (intro): The distinction between _separation_ (Wen & Liu) and _feature extraction_ (Stolk et al) is no longer prominent. This is the first section that readers encounter, and a good place for introducing the two-step structure of the example page (separation and extraction). Hence, I'd keep the original text here.

#extraction/localization: Following from the above, I'd suggest to keep my original edits here. Don't worry about "the original study", as people know what it refers to, and else they can simply click the link.

# updates: This is the section that could benefit from some more editing because you introduce concepts that people may not be familiar with, e.g. "geometric/arithmetic means", "mtmfft", "90%", "1.9", "resampling factors". I'm afraid these things are clear only you and me but not to first-time readers wanting to just get a feel for how to use IRASA and how it could help them in extracting and localizing rhythmic spectral features. So, either you introduce and describe those concepts in more detail, or you keep the updates section to a minimum in order to later add in your source IRASA pipeline. My suggestion would be the latter, and to briefly reference your github PR for the timing being as well as maybe the resampling section (though this needs some editing still to become more accessible to a general public).

Hope this helps

**Comment by ruiliu-psy** [comment]
_#IRASA (intro): The distinction between separation (Wen & Liu) and feature extraction (Stolk et al) is no longer prominent. This is the first section that readers encounter, and a good place for introducing the two-step structure of the example page (separation and extraction). Hence, I'd keep the original text here._

The simulated example also conducted 'feature extraction', as you will find in the script and the very first plot, so I don't really understand the 'distinction', however, I rephrased it, you can see if it's prominent enough now.

_#extraction/localization: Following from the above, I'd suggest to keep my original edits here. Don't worry about "the original study", as people know what it refers to, and else they can simply click the link._

I'm still reluctant to say 'the original', sorry, instead I referred to Stolk et al. 2019 again.

_#updates: This is the section that could benefit from some more editing because you introduce concepts that people may not be familiar with, e.g. "geometric/arithmetic means", "mtmfft", "90%", "1.9", "resampling factors". I'm afraid these things are clear only you and me but not to first-time readers wanting to just get a feel for how to use IRASA and how it could help them in extracting and localizing rhythmic spectral features._ 

First, these basic concepts should be covered by users before they decide to use the method and there's a link to the original paper Wen and Liu 2016. Second, you had already introduce the algorithm of IRASA at the top of the page and explained the resampling, I agreed and didn't change anything. Finally, as explained the beginning of the update section, this section was made for those who used the old implementation and were directed to this page when the run into error of their matlab code, so they should know what cfg.method='mtmfft' is. For the 'first-time readers', they can use the example script directly, I don't a conflict of the contents here.

_# So, either you introduce and describe those concepts in more detail, or you keep the updates section to a minimum in order to later add in your source IRASA pipeline._ 

That's a different thing which is not in discussion of this PR.

_#My suggestion would be the latter, and to briefly reference your github PR for the timing being as well as maybe the resampling section (though this needs some editing still to become more accessible to a general public)._

I've explained why the update section was made and why a mere link of the PR of the new implementation is insufficient, hope you understand. The last paragraph is indeed not about the update per se, but I think it should be warned because careless using filtered data can lead to serious error in the results.

**Comment by StolkArjen** [comment]
Okay, I'm fine with the spirit of this PR. There's still unnecessary clutter for my taste in the Updates section, and perhaps you'll agree at some point. ;) For now, it's fine after you've fixed the typos in one of those early sentences. I'll see if I can make a in-code note.

Btw, separation refers to IRASA, feature extraction to the identification of band-limited phenomena (e.g., sensorimotor alpha/beta).

File: example/irasa.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ tags: [example, irasa]
...
+IRASA allows distinguishing rhythmic activity from concurrent power-spectral 1/f modulations. The technique virtually compresses and expands the time-domain data with a set of non-integer resampling factors prior to Fourier-based spectral decomposition. As a result, rhythmic components in the power-spectrum are redistributed while the arrhythmic 1/f distribution is left intact. Taking the median of the resulting auto-spectral distributions extracts the power-spectral 1/f component, and the subsequent removal of the 1/f component from the original power-spectrum offers a power-spectral estimate of rhythmic content in the recorded signal.
+Below we provide two examples, a simulated dataset and real human ECoG dataset[(Stolk et al. 2019)](https://elifesciences.org/articles/48065), to demostrate [the IRASA algorithm (Wen & Liu, 2016)](https://link.springer.com/article/10.1007/s10548-015-0448-0) extracting of spectral feature.
```
[the remark this answers is not in the document]
Comment (ruiliu-psy) [reply example/irasa.md]: done

**Comment by StolkArjen** [comment]
Okay, thanks, @ruiliu-psy. Fine to pull this in, @robertoostenveld.

**Comment by robertoostenveld** [comment]
thanks for both of your contributions!

**Pull request merged**